torch/fx/experimental/symbolic_shapes.py

```diff
@@ -0,0 +1,90 @@
+import torch
+
+try:
+    import sympy
+    HAS_SYMPY = True
+except ImportError:
+    HAS_SYMPY = False
```

Comment:
This is just code motion, I hope?

Reply:
wdym? This is here so that if a file imports this without having sympy installed, it doesn't error.

Reply:
oh I wanted to know if the code here had changed

Reply:
Oh, I see. Yes, it's changed a little bit. Nick previously copied the POC from a previous version into the test file, I added an updated version into this file that we'll be building off moving forward, and modified Nick's tests to use it.